net/shorewall6/files/shorewall6.init

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
```

Comment:
Don't need a blank line here.

Also, really confused... you have the same files both in `files/` and as patches?  Why?

Reply:
Correct. Forgot to remove the file from patches.